styles/layout.less

```diff
@@ -219,13 +220,43 @@ header, #header {
     }
     #titles-preserved {
         .center-align;
-        .middle-align;
         font-size: 0.9em;
         #x-preserved {
             .bold;
             .sans-serif;
         }
     }
+    #icon-menu {
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: flex-end;
+    }
+    .icon-menu-item {
+        position: relative;
+        cursor: pointer;
+        margin-right: 1.5em;
+        .center-align;
+        text-decoration: none;
+        .sans-serif;
+        font-size: 0.7em;
+        color: @heading-color;
+        .unicolor-link(@heading-color);
+        .icon-menu-item-name {
+            font-size: 0.9em;
+        }
+        .inbox-badge {
```

Comment:
I feel that the badge is slightly lower than looks good. Any chance we can push it up a little bit?